Terrace-Svelte/src/components/Home/Weather.svelte

```diff
@@ -107,22 +35,15 @@
         sunset = data.sys.sunset;
         wind = data.wind.speed;
         timeZone = data.timezone;
-
-        document.getElementById("weather-icon").src = iconUrl;
-        document.getElementById("temperature").innerHTML = kelvinToFaren(temp) + '&deg;F';
-        document.getElementById("feels-like").innerHTML = kelvinToFaren(feelsLike) + '&deg;F';
-        document.getElementById("humidity").innerHTML = humidity + "%";
-        document.getElementById("wind-speed").innerHTML = wind + "MPH";
-        document.getElementById("sunrise").innerHTML = epochToReadable(sunrise) + "AM";
-        document.getElementById("sunset").innerHTML = epochToReadable(sunset) + "PM";
-    }
-    // Fix this to use svelte window syntax
-    if (document.readyState !== 'loading') {
-        initCode();
-    } else {
-        document.addEventListener('DOMContentLoaded', function () {
-            initCode();
-        });
+//Math.round(1.8 * (k - 273) + 32)
+        elmWeatherIcon = icon;
+        elmWeatherIconSrc = iconUrl;
+        elmTemperature = Math.round(1.8 * (temp - 273) + 32) + '&deg;F';
+        elmFeelsLike = Math.round(1.8 * (feelsLike - 273) + 32) + '&deg;F';
+        elmHumidity = humidity + "%";
+        elmWindSpeed = wind + "MPH";
+        elmSunrise = new Date(sunrise * 1000).toLocaleTimeString(undefined, { hour: '2-digit', minute: '2-digit', hour12: true });
+        elmSunset = new Date(sunset * 1000).toLocaleTimeString(undefined, { hour: '2-digit', minute: '2-digit', hour12: true });
```

Comment:
moved to using a more sveltey way of doing this. did some date string condensing